dataland-frontend/src/components/forms/parts/fields/NaceCodeFormField.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="form-field">
     <UploadFormHeader :label="label" :description="description" :is-required="required" />
-    <NaceCodeSelector v-model="selectedNaceCodes" :shouldDisableCheckboxes="shouldDisableCheckboxes" />
+    <NaceCodeSelector
+      v-model="selectedNaceCodes"
+      :shouldDisableCheckboxes="shouldDisableCheckboxes"
+      :valueTree="valueTree"
```

Comment:
Why are there changes to the Nace Code Component?

Reply:
@MarcinStanczyk7 should know why.

Reply:
So that the NaceCode component can be used with other data than just Nace Codes. However, since we are not using NaceCodeComponent in DataRequest page but a Multiselect. There is no need to extend the NaceCodes component with this capability, so I will remove the code until it is needed.

Reply:
ok